coldfront/core/user/templates/account_deactivation/reason_card.html

```diff
@@ -0,0 +1,13 @@
+<div class="card">
+  <div class="card-header"><strong>Reason Legend</strong></div>
+  <div class="card-body">
```

Comment:
These descriptions should probably be pulled from the database (the typo mentioned above is here too).

(The 1 and 2 in `request_list_table.html` would have to be consistent with what's displayed here. Maybe write a method that gets the reasons in ascending PK order, and then assign 1, 2, etc. to them.)